index.bs

```diff
@@ -169,22 +169,32 @@ Note: Sanitizing a string will use the [=HTML Parser=] to parse the input,
 ## The Configuration Dictionary {#config}
 
 The <dfn lt="configuration">sanitizer's configuration object</dfn> is a
-dictionary which describes modifications to the sanitze operation.
+dictionary which describes modifications to the sanitze operation. If a
+Sanitizer has not received an explicit configuration, for example when being
+constructed without any parameters, then the [=default configuration=] value
+is used as the configuration object.
 
 <pre class="idl">
   dictionary SanitizerConfig {
     sequence&lt;DOMString> allowElements;
+    sequence&lt;DOMString> allowContentElements;
     sequence&lt;DOMString> blockElements;
     sequence&lt;DOMString> dropElements;
     AttributeMatchList allowAttributes;
     AttributeMatchList dropAttributes;
+    boolean allowUnknownElements;
+    boolean allowUnknownAttributes;
```

Comment:
I don't want to turn this into endless bikeshedding, but I think a naming scheme where sequences and boolean flags are both named `allow...` is confusing. I don't have a better suggestion as of yet, but would like to address this somehow.

Reply:
Hmm. I can't come up with a super good name. If we remove the allowUnknown* options, the issue becomes moot, though.